zerver/webhooks/flock/doc.md

```diff
@@ -0,0 +1,24 @@
+Get Zulip notifications from your Flock channels.
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to **Apps** in the bottom-right corner.
+
+1. Then, Go to **Admin Panel** -> **Webhooks**.
+
+1. Click on the **Add** button next to **Outgoing Webhook**.
```

Comment:
Actually, replace with `Next to **Outgoing Webhook**, click on **Add**.` Sounds cleaner! :)